src/sparql/cl-def-xrefs.sparql

```diff
@@ -0,0 +1,14 @@
+prefix oio: <http://www.geneontology.org/formats/oboInOwl#>
+prefix owl: <http://www.w3.org/2002/07/owl#>
+PREFIX definition: <http://purl.obolibrary.org/obo/IAO_0000115>
+
+SELECT ?cls ?xref WHERE 
+{
+  ?cls definition: ?def .
+  ?ax a owl:Axiom;
+    owl:annotatedSource ?cls;
+    owl:annotatedProperty definition:;
+    owl:annotatedTarget ?def;
+    oio:hasDbXref ?xref .
+    FILTER(isIRI(?cls) && (STRSTARTS(str(?cls), "http://purl.obolibrary.org/obo/CP_") || STRSTARTS(str(?cls), "http://purl.obolibrary.org/obo/CL_") || STRSTARTS(str(?cls), "http://purl.obolibrary.org/obo/cl#")))
+}
```

Comment:
Sure you want to include deprecated classes?

Reply:
Thanks for the reminder!

Reply:
done